ports/battalion/port.json

```
@@ -0,0 +1,41 @@
+{
+    "version": 3,
+    "name": "battalion.zip",
+    "items": [
+        "BATTALION.sh",
```

```suggestion
        "Battalion.sh",
```
Your shell script is not in all caps.  You can change the shell script to be in all caps or you'll need to change the entries in your gameinfo.xml and port.json as per my suggestions to match the shell script name especially with case sensitivity.